### PR TITLE
#238 Delete Widget Functionality

### LINKED ISSUE #232: The date of calendar widget should match date of the page
**Describe the bug**
The date of calendar widget does not match date of the page.

**To Reproduce**
1. Click on the -> on the journal a few times to make sure the date is in the future
2. Click on the calendar icon to add a calendar widget
3. See that the dates don't line up

**Expected behavior**
The date of calendar widget should match date of the page.

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: Windows 10 v. 2004
 - Browser Firefox
 - Version 87.0 (64-bit)

**Additional context**
N/A

### LINKED ISSUE #238: Delete journal widgets
**Is your feature request related to a problem? Please describe.**
Currently there is an issue that widgets can only be added, and not removed.

**Describe the solution you'd like**
The ability to remove widgets either through an icon in the widget, or a context menu. The user should be prompted to ensure they meant the action.

**Describe alternatives you've considered**
Drag-and-drop trash can which could work well however the UI likely doesn't have enough space to accommodate.

**Additional context**
This is a sub-issue of #65

### PR DESCRIPTION
**Pull request by OverCry**
Issue #238 

## Proposed Changes

- Added two packages for styling
- Added an option to delete widgets
- Reflected this change onto the local database
- Refactored JournalCalander to keep consistent code style

### CODE REVIEW

**Review by David-Xia0 — DISMISSED** [review]
You are merging into the wrong main branch, please double check

**Comment by OverCry** [comment]
> You are merging into the wrong main branch, please double check

Oh dang right that completely slipped my mind. Ty for checking

**Comment by OverCry** [comment]
Changed to draft to resolve conflicts first

File: telos-frontend/src/dnd/DraggableWidget.jsx [hunk 1]
```diff
@@ -13,8 +14,8 @@ function getStyles(left, top) {
 }
 
 // A component for widgets that can be dragged
-const DraggableWidget = memo((props) => {
-  const { id, position, type, data, date } = props;
+const DraggableWidget = memo(({ id, position, type, data, date, deleteWidget }) => {
+  // const { id, position, type, data, date, update } = props;
```
Comment (jemimahmartinez) [inline telos-frontend/src/dnd/DraggableWidget.jsx]: Could delete this commented line since you refactored the code?

**Review by jemimahmartinez — APPROVED** [review]
Awesome stuff! The flow is nice and smooth, well done 👌

File: telos-frontend/src/dnd/DraggableWidget.jsx (hunk 1 above)
Comment (OverCry) [reply telos-frontend/src/dnd/DraggableWidget.jsx]: Oh yep i'll quickly remove it now

**Review by ayushjani — APPROVED** [review]
Great job! Looks and works well 👌

**Pull request merged**